src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] getColor = Color.values();
+        int index = new Random().nextInt(getColor.length);
+        return getColor[index].name();
```

Comment:
1. getColor is a good name for method, but not for variable.
2. Move Random to class field. Don't create new Random each time this method is invoked.
```suggestion
        Color[] colorValues = Color.values();
        int index = random.nextInt(colorValues.length);
        return getColor[index].name();
```